src/utils.ts

```diff
@@ -35,7 +35,7 @@ export function walk(cwd: string, dir = cwd, files: string[] = []) {
 		}
 	});
 
-	return files;
+	return files.filter(v => !v.includes('.DS_Store'));
```

Comment:
Instead of filtering the array after we have just created it, could we just prevent the addition of `.DS_Store` files to the array in the first place? Additionally, comparing only the `file` value will gives us a bit more certainty that we are only omitting `.DS_Store` files and nothing else. 

False positives seem unlikely, but I've stopped trying to second-guess all of the weird things people do with Sapper.

Reply:
yeah, I agree that seems a bit better. I've tweaked the PR to do that